src/App.js

```diff
@@ -31,7 +31,8 @@ function App() {
       <div className="App-header">
         <div className="App-container">
           <div className="App-section">
-            <h1 className="text-xl">Michael Bloomberg would pay $3.163 billion next year under Elizabeth’s wealth tax.</h1>
+            <title>How Elizabeth Warren Would Make Bloomberg Pay Your Student Loans</title>
+            <h2 className="text-lg">Michael Bloomberg has proposed a $700 million plan to tackle student debt. But under Elizabeth Warren’s <a href="https://elizabethwarren.com/plans/ultra-millionaire-tax">Ultra-Millionaire Tax</a> plan, Bloomberg would pay <a href="https://elizabethwarren.com/calculator/ultra-millionaire-tax">$3.16 billion</a>next year alone.</h2>
```

Comment:
> Michael Bloomberg has proposed a $700 million plan to tackle student debt

Is this fair? After 20 years (ha!) they'd get forgiven, so the total plan would presumably be more.

I'm not sure we want to highlight his plan in any way / risk getting into an argument about it or having to update this text when he updates his plan. He can _say_ anything

Reply:
If we want to make a specific dig - my initial thought is to focus on the amount of money he's trying to spend to buy the election!

That's unequivocal and unlikely to change. Though I agree less relevant.

I just don't want to have a moving target, or having this be a jumping off point for a discussion of _Bloomberg's_ plan in any way

Reply:
I hear you! The reason I brought up the plan is that people could accuse us of being misleading we use him as an example but didn't acknowledge that TECHNICALLY he also does have a plan for student debt. Sigh.

But I agree that we want to keep laser-focused on the ultra-millionaire tax (not Bloomberg's ideas). Maybe we can do this by focusing on him not PERSONALLY paying off anyone's student loans? I mean, if I had that kind of cheddar I would be paying off every person's debt I could get my hands on...

How's this?

Michael Bloomberg has spent over $500 million of his own funds on his campaign. Even with $60 billion to spare, he hasn't (yet) offered to personally pay off any student loan debt.

Under Elizabeth Warren’s <a href="https://elizabethwarren.com/plans/ultra-millionaire-tax">Ultra-Millionaire Tax</a> plan, Bloomberg would pay <a href="https://elizabethwarren.com/calculator/ultra-millionaire-tax">$3.16 billion</a> in taxes next year alone. [.. etc]

Reply:
😅 Idk this is feeling like a weird flex to me. Let’s chat in the channel